nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_kernel.py

```diff
@@ -206,7 +215,7 @@ def generate_kernel_size_list(self, max_kernel_size: KernelSizeType) -> List[Ker
             return [1]
         kernel = max_kernel_size
         ks_list = []
-        while kernel > 1:
+        while kernel >= max(max_kernel_size - 2 * original_padding_value, 3):
```

Comment:
Using max(max_kernel_size - 2 * original_padding_value, 3) instead of max_kernel_size - 2 * original_padding_value is mainly for consistency with the old code. The old code doesn't allow 1 to appear in elastic kernel lists.